### PR TITLE
Add VirtualBox version 6.1.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20213)

### CODE REVIEW

File: manifests/o/Oracle/VirtualBox/6.1.0/Oracle.VirtualBox.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+PackageIdentifier: Oracle.VirtualBox
+PackageName: Oracle VM VirtualBox
+Moniker: virtualbox
+PackageVersion: 6.1.0
+Publisher: Oracle Corporation
+Author: Oracle Corporation
+License: (c) 2020 Oracle Corporation
+LicenseUrl: https://www.virtualbox.org/wiki/Licensing_FAQ
+ShortDescription: Oracle.VirtualBox
```
Comment (ImJoakim) [inline manifests/o/Oracle/VirtualBox/6.1.0/Oracle.VirtualBox.yaml:9]:
```
ShortDescription: Oracle.VirtualBox
```
Can you please add a ShortDescription 😃

**Comment by ImJoakim** [comment]
Please take a look at my comment above 😃 

- https://github.com/microsoft/winget-pkgs/pull/20213#discussion_r666288465

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/20213#pullrequestreview-702193667

**Comment by vedantmgoyal2009** [comment]
@ItzLevvie @ImJoakim Done

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**